docs/guide/time.rst

```diff
@@ -32,24 +66,64 @@ Here are few examples of formats which `sunpy.time.parse_time()` accepts::
     >>> parse_time('20070504_210812')
     <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
 
-Each of the above returns the same `~astropy.time.Time` object ``<Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>``. One of the most standard time formats used in solar
-physics is the number of seconds since 1979 January 01 called ``utime``. The parse_time
-function also accepts this as input, e.g.::
+Each of the above returns the same `~astropy.time.Time` object ``<Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>``.
+
+One of the most common time formats used in unix is the number of seconds since 1979 January 01
+called ``utime``. The parse_time function also accepts this as input, e.g.::
 
     >>> parse_time(894316092.00000000, format='utime')
     <Time object: scale='utc' format='utime' value=894316092.0>
 
+Other formats
+^^^^^^^^^^^^^
+
+`sunpy.time.parse_time()` understands more than just a string.
+For example::
+
+    >>> import datetime
+    >>> parse_time(datetime.datetime(2007, 5, 4, 21, 8, 12))  # datetime.datetime
+    <Time object: scale='utc' format='datetime' value=2007-05-04 21:08:12>
+    >>> parse_time(datetime.date(2007, 5, 4))  # datetime.date
+    <Time object: scale='utc' format='iso' value=2007-05-04 00:00:00.000>
+
+    >>> parse_time((2007, 5, 4, 21, 8, 12))  # tuple of numbers
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
+
+    >>> import pandas
+    >>> parse_time(pandas.Timestamp('2007-05-04T21:08:12'))  # pandas.Timestamp
+    <Time object: scale='utc' format='datetime' value=2007-05-04 21:08:12>
+
+    >>> time_ranges = [datetime.datetime(2007, 5, i) for i in range(1, 3)]
+    >>> parse_time(pandas.Series(time_ranges))  # pandas.Series
+    <Time object: scale='utc' format='datetime' value=[datetime.datetime(2007, 5, 1, 0, 0) datetime.datetime(2007, 5, 2, 0, 0)]>
+
+    >>> parse_time(pandas.DatetimeIndex(time_ranges))  # pandas.DatetimeIndex
+    <Time object: scale='utc' format='datetime' value=[datetime.datetime(2007, 5, 1, 0, 0) datetime.datetime(2007, 5, 2, 0, 0)]>
+
+    >>> import numpy as np
```

Comment:
I would put all the imports at the top.

Reply:
I wanted to seperate the examples out by module.